keras_core/backend/common/variables.py

```diff
@@ -408,7 +408,7 @@ def standardize_dtype(dtype):
             dtype = "int32"
     if hasattr(dtype, "name"):
         dtype = dtype.name
-    elif config.backend() == "torch":
+    if hasattr(dtype, "__str__") and "torch" in str(dtype):
```

Comment:
You can use elif for better performance (and above too)

Reply:
Done.